Dockerfile

```diff
@@ -101,8 +101,9 @@ USER redash
 
 # Customized runtime environment variables
 ENV REDASH_WEB_WORKERS=8
+ENV WORKER_COUNT=3
 ENV KEEP_ALIVE=305
-ENV LOG_LEVEL=debug
+ENV LOG_LEVEL="info"
```

Comment:
I've currently set the log level to warning, but no real preference from my end

Reply:
It does give you a few more tidbits about processes restarting, but happy to reduce even further.